source/templates/classic/image.svg

```diff
@@ -897,12 +897,12 @@
 
       <% if (base.metadata) { %>
         <footer>
-          <span>These metrics <%= !computed.token.scopes.includes("repo") ? "does not include all" : "includes" %> private contributions<% if ((config.timezone?.name)&&(!config.timezone?.error)) { %>, timezone <%= config.timezone.name %><% } %></span>
+          <span>These metrics <%= !computed.token.scopes.includes("repo") ? "do not include all" : "include" %> private contributions<% if ((config.timezone?.name)&&(!config.timezone?.error)) { %>, timezone <%= config.timezone.name %><% } %></span>
```

Comment:
The timezone transclusion seems a bit misplaced but not sure what to do with it.

Reply:
Yeah, I think it's beceause it's in italic so it get cropped but's that's ok, I planned to add more padding too the right later 👍